### PR TITLE
Bugfix for builddep, repoquery and download plugin (RhBug: 1186381) (RhBug: 1225784) (rhBug:1250114)

### PR DESCRIPTION
**Pull request by max9631**

### CODE REVIEW

**Comment by mluscon** [comment]
ok to test

File: plugins/builddep.py [hunk 1]
```diff
@@ -121,13 +121,22 @@ def run(self, args):
...
-            else:
-                self._remote_deps(pkgspec)
+            try:
+                if pkgspec.endswith('.src.rpm') or pkgspec.endswith('nosrc.rpm'):
+                    self._src_deps(pkgspec)
+                elif pkgspec.endswith('.spec'):
+                    self._spec_deps(pkgspec)
+                else:
+                    self._remote_deps(pkgspec)
+            except Exception, e:
```
Comment (jsilhan) [inline plugins/builddep.py]: Could be the `Exception` more specific?

**Comment by mluscon** [comment]
Please rebase yor commits on the current upstream master. Is #ab00289 still applicable?

**Comment by mluscon** [comment]
Merged, thank you.

**Comment by max9631** [comment]
Thanks for your patience :)

**Pull request closed without merging**